### DISCUSSION TITLE
Switch leaderboard category button arrangement

### DISCUSSION THREAD

**Discussion by zvyap** [ppy/osu#26176] [Q&A] [answered]
I had opened pull request #26163, which was closed due to the screen being redesigned. I fully understand why it was closed: the developers don't want to waste their effort on a screen that is going to be redesigned.

Change I mentioned:
![image](https://github.com/ppy/osu/assets/52874570/ce5f3b52-085d-4785-9267-6c00cd04a81f)
To This:
![image](https://github.com/ppy/osu/assets/52874570/3b29335e-6db6-4b36-a862-f60f8065d792)

However, it's a very simple change that would offer players a lot of convenience. 
Although it is temporary, it's still a good change for players if osu! wants to ensure new users get a good first impression of Lazer.
(Alt+Tab two times for changing frequently leaderboard category is really annoying)

**Comment by peppy** [comment, marked as answer]
https://github.com/ppy/osu/pull/26177
